core-web/libs/sdk/experiments/package.json

```diff
@@ -1,26 +1,24 @@
 {
-  "name": "@dotcms/experiments",
-  "version": "0.0.1-alpha",
-  "dependencies": {},
-  "description": "Official JavaScript library to use Experiments with DotCMS.",
-  "type": "module",
-  "main": "./index.cjs",
-  "module": "./index.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/dotCMS/core.git#master"
-  },
-  "keywords": [
-    "dotCMS",
-    "CMS",
-    "Content Management",
-    "A/B testing",
-    "Experiments"
-  ],
-  "author": "dotcms <dev@dotcms.com>",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/dotCMS/core/issues"
-  },
-  "homepage": "https://github.com/dotCMS/core/tree/master/core-web/libs/sdk/experiments/README.md"
+    "name": "@dotcms/experiments",
+    "version": "0.0.1-alpha",
+    "description": "Official JavaScript library to use Experiments with DotCMS.",
+    "dependencies": {},
+    "main": "./dot-experiments.iife.js",
```

Comment:
The example I am following is to use this as plain HTML, without complicating things, and leave them as they use it in dotCMS currently in Experiments is an IIFE, in fact speaking with Rodriguez, we agreed that when this is working in IIFE it will replace what he did this, and that way we will only maintain 1 JS code, not the one he generates in JSP but @dotcms/experiment.
The other thing, when we are working on NextJS or another framework I will expose the SDKExperiment class, currently I haven't gotten around to that, but it's the idea.

What do you think?

Reply:
I put TODO in the code, the standalone we will create the file and move to `dotCMS/src/main/webapp/html` and the entry o main file will be sdkExperiment